app/javascript/src/lib/dayjs.js

```diff
@@ -0,0 +1,5 @@
+// in src/utils.js or at the closest scope
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+dayjs.extend(relativeTime);
```

Comment:
You should be default exporting the dayjs instance. Otherwise it's possible that either VSCode or Eslint might complain when we import this module.

Reply:
There were no error suggestions while using this. Anyway, I have default exported this now.